resources/features/Stratification.feature

```diff
@@ -0,0 +1,69 @@
+              | 1100000004 | 9906000145 | AUBASOT(CHRISTCHURCH) LIMITED |              | 1 GARTH EDGE             | SHAWFORTH      | WHITWORTH   | ROCHDALE LANCASHIRE     |          | OL12 8EH | 2           | 45320 | 0           | 0         | 7            | 7            | 0            | 0             | 0            | 0.509298879 |
+              | 1100000005 | 9906000175 | HIBAER                        |              | 1 GEORGE SQUARE          | GLASGOW        |             |                         |          | G2 5LL   | 1           | 45400 | 1           | 1         | 106          | 106          | 0            | 0             | 0            | 0.147768898 |
+              | 1100000006 | 9906000205 | HIBAER                        |              | 1 GLEN ROAD              | HINDHEAD       | SURREY      |                         |          | GU26 6QE | 1           | 46110 | 1           | 1         | 297          | 297          | 0            | 0             | 0            | 0.588701588 |
+              | 1100000007 | 9906000275 | IBANOCTRACTS UK LTD           |              | 1 GLYNDE PLACE           | HORSHAM        | WEST SUSSEX |                         |          | RH12 1NZ | 1           | 46120 | 2           | 2         | 287          | 287          | 0            | 0             | 0            | 0.155647458 |
+              | 1100000008 | 9906000325 | TLUBARE                       |              | 1 GORSE ROAD             | REYDON         | SOUTHWOLD   |                         |          | IP18 6NQ | 1           | 46130 | 3           | 3         | 197          | 197          | 0            | 0             | 0            | 0.446872271 |
+              | 1100000009 | 9906000355 | BUCARR                        |              | 1 GRANVILLE AVENUE       | LONG EATON     | NOTTINGHAM  |                         |          | NG10 4HA | 1           | 46140 | 1           | 1         | 18           | 18           | 0            | 0             | 0            | 0.847311602 |
+              | 1100000010 | 9906000405 | DCAJ&WALTON                   |              | 1 GRANVILLE AVENUE       | LONG EATON     | NOTTINGHAM  |                         |          | NG10 4HA | 1           | 46150 | 2           | 2         | 72           | 72           | 0            | 0             | 0            | 0.548604086 |
+              | 1100000011 | 9906000415 | &BAMCFLINT                    |              | 1 GARENDON WAY           | GROBY          | LEICESTER   |                         |          | LE6 0YR  | 1           | 46160 | 1           | 0         | 400          | 400          | 0            | 0             | 0            | 0.269071541 |
+          And a Stratification Properties file that does not exist
+          When an exception in <language> is thrown for Stratified Properties not being found upon trying to Stratify
+
+          @JVM
+          Examples:
+          | language |
+          | Scala    |
```

Comment:
New Scenario required to cover units in a given frame not being referenced by the Stratification Properties i.e. no cell allocation.
The response should be that they are not allocated a cell_no (strata number left blank) but are included in the output csv (all units on the input frame should be in the output frame) #with an error stating units in Frame not allocated to strata logged.

Reply:
May need more clarity on this one